pocs/camera/camera.py

```diff
@@ -423,25 +418,31 @@ def _setup_observation(self, observation, headers, filename, **kwargs):
 
         start_time = headers.get('start_time', current_time(flatten=True))
 
+        if not observation.seq_time:
```

Comment:
FWIW, it seems to me that cameras should have no knowledge of observations; e.g. a camera should get asked to take one image and put it in a specific file, nothing more.

Reply:
To some extent I agree and think this could be re-worked, but this is part of the `take_observation` method, which necessitates an `observation` be present. The lower-level `take_exposure` method does not require this knowledge and instead requires a filename be passed.

Reply:
Perhaps a TODO?

Reply:
Issue added here: https://github.com/panoptes/POCS/issues/656